FluentTc.Tests/AcceptanceTests.cs

```diff
@@ -221,13 +221,13 @@ public void RunBuildConfiguration_WithComment()
             var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller, buildConfigurationRetriever);
 
             // Act
-            connectedTc.RunBuildConfiguration(having, options => options.WithComment("comment!"));
+            var build = connectedTc.RunBuildConfiguration(having, options => options.WithComment("comment!"));
```

Comment:
@nex-54 that's great that RunBuildConfiguration has a return value. Now you can assert on it.